src/aspire/utils/misc.py

```diff
@@ -170,15 +170,15 @@ def gaussian_2d(size, mu=(0, 0), sigma=(1, 1), indexing="yx", dtype=np.float64):
     # Construct centered mesh
     g = grid_2d(size, shifted=False, normalized=False, indexing=indexing, dtype=dtype)
 
+    X, Y = g["x"], g["y"]
```

Comment:
Is this just cosmetic or is there a deeper reason here?

Reply:
Avoids mutating the dictionary performing the swap.